frontend/src/App.tsx

```diff
@@ -1,24 +1,23 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import axios from "axios";
+import { useState } from "react";
+const serverURL = 'http://localhost:8000'
 
 function App() {
+  const [message, setMessage] = useState("");
+  
+  const handleClick = async () => {
+    if (message) {
+      setMessage("")
+      return
+    }
+    const res = await axios.get(`${serverURL}/api`)
+    const serverMessage = res.data.data
```

Comment:
not a necessity for this temporary code but it's good to think about how you would handle this route if it ever errored, since we are assuming in this code so far that the API always successfully returns

https://stackabuse.com/handling-errors-with-axios/

Reply:
Ok, I added a try and catch block to catch network errors:

    try {
      const res = await axios.get(`${serverURL}/api`)
      const serverMessage = res.data.data
      setMessage(serverMessage)
    } catch (error) {
      //error has type 'unknown'. My fix: assert error as any.
      console.error("Error connecting to backend:", (error as any).message)
      alert(`Error connecting to backend: ${(error as any).message}`)
    }